extra_docs/ums/ums_draft.md

```diff
@@ -92,21 +92,18 @@ ums.activate({
 });
 ```
 
-### Get user by id
+### Fetch user by auth alias (or email)
 ```javascript
-ums.getUserById(userId); // -> return user or throw an error
```

Comment:
![Elon Mask](https://preview.redd.it/yc1n5m9y1sf21.png?width=960&crop=smart&auto=webp&s=2aa0539dde06eff0374e50069c31ef708ac7984d)

Reply:
:-D